translations/en-us.yaml

```diff
@@ -1003,6 +1003,7 @@ banner:
   logging: We've made big improvements to logging in <a href="{dashboardLink}/logging">Cluster Explorer</a>.  See <a href="{docsBase}">docs</a> for details on what's new and how to migrate.
   multiclusterapp: We've updated deploying apps at scale with Fleet in Cluster Explorer.
   monitoring: We've made big improvements to monitoring in <a href="{dashboardLink}/monitoring">Cluster Explorer</a>.  See <a href="{docsBase}">docs</a> for details on what's new and how to migrate.
+  monitoringV2Warning: Monitoring is currently deployed in <a href="{dashboardLink}/monitoring">Cluster Explorer</a> and needs to be removed through <a href="{dashboardLink}/monitoring">Cluster Explorer</a> to enable monitoring Rancher.
```

Comment:
@vincent99 can we link to the installed apps page?

Reply:
I think I could just change `/monitoring` with `/apps/catalog.cattle.io.app`?